benchmarks/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
I've opted for a bash script just to avoid polluting `package.json` with dependencies that are only used in the benchmark fixtures, but I'm open to alternative ideas.